### ISSUE TITLE
Bump exiv2 package version in Ubuntu 22.04 from 0.27.5 to 0.28.0

### ISSUE THREAD

**Issue by Ryanf55** [Exiv2/exiv2#2627]
#### **Describe the solution you would like**

When installing `exiv2` on Ubuntu 22.04, I would like to have the `0.28.0` version installed, which corresponds to the latest release. There are features not available in `0.27.5` 

#### **Describe alternatives you have considered**

Modify my build system and build from source -> This is time intensive

#### **Desktop**
- OS and version: Ubuntu 22.04
- Exiv2 version and source: 0.28.0
- Custom

**Comment by kmilos** [comment]
Please address this to Ubuntu, the Exiv2 project has no influence over the various distributions.

**Comment by Ryanf55** [comment]
Thanks. I just sent an request to the debian maintainers list.

**Comment by Ryanf55** [comment]
Hello, I have not received any replies from the debian maintainers list. I sent the email to `exiv2@packages.debian.org` and got this autoreply from `pkg-kde-extras-owner@alioth-lists.debian.net`
Message I sent:
```
I'd like to request a version bump of the exiv2 package available in Debian. I use Ubuntu 22.04, which [apparently follows the debian versions](https://answers.launchpad.net/ubuntu/+source/exiv2/+question/700398). 

Current debian buster version[: 0.27.3](https://packages.debian.org/stable/exiv2)
Requested:[ 0.28.0](https://github.com/Exiv2/exiv2/releases/tag/v0.28.0)

Related Issues:
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1000788
https://github.com/Exiv2/exiv2/issues/2627

Why the change? 
The 0.28.0 version has a modernized C++ interface available, which was added in this [PR#1555](https://github.com/Exiv2/exiv2/pull/1555).

Please let me know if this would be possible. 

Thank you!
```

I got this reply. Seems like it was never approved or rejected.

```
Your mail to 'pkg-kde-extras' with the subject

    Request to bump exiv2 to 0.28.0

Is being held until the list moderator can review it for approval.

The reason it is being held:

    Post by non-member to a members-only list
```

Can anyone recommend a different method that I could achieve a better response?

**Comment by kmilos** [comment]
A gentle reminder that this is a tracker **_for actual Exiv2 issues_**, please do not abuse developer time.

That being said, given the usual conservatism of Debain/Ubuntu maintainers and the time it usually takes for things to percolate there, [with the sheer amount of Exiv2 dependants that also need to be patched/updated alongside because of the 0.28 API change](https://archlinux.org/todo/exiv2-028/), I personally don't think this is going to happen until Debian 13.

So you'll just have to build your own package if the binary provided at the above link doesn't work, or switch to a bleeding edge/rolling distro. @Ryanf55
